dts/st/f0/stm32f030r8tx-pinctrl.dtsi

```diff
@@ -496,7 +496,7 @@
 			};
 
 			/omit-if-no-ref/ tim3_ch3_pc8: tim3_ch3_pc8 {
-				pinmux = <STM32_PINMUX('C', 8, AF0)>;
+				pinmux = <STM32_PINMUX('C', 8, AF1)>;
```

Comment:
Previous code was correct. The datasheet agrees that it is AF0, and not AF1.
The issue has been raised on CubeMX teams side.